src/routing/http-routing-v1.md

```diff
@@ -106,6 +106,60 @@ Each object in the `Providers` list is a record conforming to a schema, usually
 
 ## Peer Routing API
 
+### `GET /routing/v1/closest-peers/{peer-id}?[closerThan]&[count]`
```

Comment:
I'm not sure this makes sense at the `/routing/v1` namespace since it's fairly specific to systems like the Amino DHT.

Having an HTTP API for this kind of thing makes sense though, perhaps as `/amino` or `/kad` or something. It could be nested under `/routing/v1/<amino, etc.>` as well if that makes folks happier.
- Note: Given that this is quite close to being the HTTP equivalent of the `FIND_NODE` RPC something like this could potentially end up being worked into that protocol itself (e.g. to enable http-based request/response).

Reply:
My hope was that we could have it be routing-agnostic but maybe that's not possible, since the actual results would very much depend on what was being queried and how.

I'm happy with:

`GET /routing/v1/amino/closest-peers/{peer-id}?[closerThan]&[count]`

though if it's all amino, perhaps we should just have done with it and use the RPC names?

- `GET /routing/v1/amino/find-node/{multibase}`

...then maybe later:

- `GET /routing/v1/amino/get-value/{multibase}`
- `GET /routing/v1/amino/get-providers/{cid}`

though the second is redundant given the existing HTTP endpoints and the first is only partially redundant since we could use it to search for more than just IPNS records.

...and even later `PUT_VALUE` and `ADD_PROVIDER`?